Lorem Ipsum XR Translator/Assets/Scripts/SpeechToText/SpeechToTextDisplay.cs

```diff
@@ -0,0 +1,177 @@
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+using System.Collections;
+using Translator;
+using Config;  
+
+namespace STT
+{
+    /// <summary>
+    /// The class SpeechToTextDisplay is defined within the STT namespace is used to translate and display in the scene 
+    /// </summary>
+    public class SpeechToTextDisplay : MonoBehaviour
+    {
+        /// <summary>
+        /// Calls the AzureSpeechToText script
+        /// </summary>
+        [SerializeField]
+        private AzureSpeechToText azureSpeechToText; 
```

Comment:
Using [SerializeField] instead of public is best practice. Good job! Proud of you :P